GitExtUtils/GitUI/ImageListExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Runtime.InteropServices;
+
+namespace GitUI
```

Comment:
```suggestion
namespace GitUI;
```

Reply:
❔ for the const and import:
```suggestion
namespace System;

internal static partial class NativeMethods
```

Reply:
@mstv please have a look if I understood you correctly. I've moved the interop method (and a constant) into `GitExtUtils/GitUI/Interops/ComCtl32/ImageListSetBkColor.cs` following a pattern used in `GitUI` project.